bears/general/GixyBear.py

```diff
@@ -0,0 +1,10 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+
+@linter(executable='gixy',
+        output_format='regex',
```

Comment:
this only says which mode the bear is in.  you need to write the regex.  see the linter bear docs.  it has a tutorial which covers this.